Src/AutoFixtureUnitTest/FixtureTest.cs

```diff
@@ -948,8 +948,8 @@ public void InjectCustomUpperLimitWillCauseSutToReturnNumbersInLimit()
             var result = sut.Create<DoublePropertyHolder<int, long>>();
             // Verify outcome
             Assert.True(
-                (result.Property1 >= lower && result.Property1 < upper) &&
-                (result.Property2 >= lower && result.Property2 < upper)
+                (result.Property1 >= lower && result.Property1 <= upper) &&
+                (result.Property2 >= lower && result.Property2 <= upper)

```

Comment:
OK, thank you, @Pvlerick and @jryanmoseley, at this point I think you understand this better than I do :flushed: